src/webview/js/js.js

```diff
@@ -593,6 +593,14 @@ documentBody.addEventListener('click', (e: MouseEvent) => {
     return;
   }
 
+  if (target.parentNode instanceof Element && target.parentNode.matches('.header')) {
```

Comment:
Is this change required only because you've added a `div` inside the '.header'?

Try using CSS: `pointer-events: none;` for that `div` and it will stop handling click/touch events.

Reply:
Done! Thanks for the advice! 🙂